### ISSUE TITLE
Reverse dependencies?

### ISSUE THREAD

**Issue by sckott** [metacran/crandb#37]
Seems like these aren't collected, or am I missing something. perhaps there is a route that has these e.g., like `/pkg/deps`

**Comment by gaborcsardi** [comment]
Yes, updating them is not trivial, that's why it is not there. It is not hard of course, but I need to write some external JS code to do it, because the DB is not very good at these sort of things.

**Comment by sckott** [comment]
okay, close if you want

**Comment by gaborcsardi** [comment]
No, I want to implement them, so let's leave this open.

On Thu, Nov 12, 2015 at 7:46 PM, Scott Chamberlain <notifications@github.com
